src/modules/35-connectors/components/CreateConnector/commonSteps/ConnectorDetailsStep.tsx

```diff
@@ -155,7 +155,7 @@ const ConnectorDetailsStep: React.FC<StepProps<ConnectorConfigDTO> & ConnectorDe
             ...props.formData
           }}
         >
-          {formikProps => {
+          {(formikProps: any) => {
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232293)
```suggestion
          {(formikProps: unknown) => {
```